### DISCUSSION TITLE
pint does not map bar/Pa to unitless.

### DISCUSSION THREAD

**Discussion by SullivanMuse** [hgrecco/pint#1589] [Q&A] [answered]
Why does `ureg.bar/ureg.Pa` not give answer as `10**5`?

Shouldn't this be unitless?

If not, please explain where I am going wrong.

**Comment by hgrecco** [comment, marked as answer]
It does if you use `auto_reduce_dimensions=True`, take a look at the docs.
or do it in an explicity way.
```python
>>> (1.*ureg.bar/ureg.Pa).to_base_units()
<Quantity(100000.0, 'dimensionless')>
```
